lib/galaxy/managers/base.py

```diff
@@ -736,7 +736,7 @@ class ModelDeserializer( HasAModelManager ):
     """
     # TODO:?? a larger question is: which should be first? Deserialize then validate - or - validate then deserialize?
 
-    def __init__( self, app, **kwargs ):
+    def __init__( self, app, validator=None, **kwargs ):
```

Comment:
The new validator parameter does not seem to be used, is it for future functionalities?